layouts/partials/fragments/contact.html

```diff
@@ -33,71 +33,71 @@ <h2>{{- . | markdownify -}}</h2>
     </div>
     <div class="row">
       <div class="col-12">
-        {{ if not $data.postURL }}
-          <form class="contact" action="https://formspree.io/{{ $data.email }}" method="POST" name="sentMessage" id="{{ $data.formName | default "contactForm" }}">
+        {{ if not $contact.postURL }}
+          <form class="contact" action="https://formspree.io/{{ $contact.email }}" method="POST" name="sentMessage" id="{{ $contact.formName | default "contactForm" }}">
         {{ else }}
-          <form class="contact" action="{{ $data.postURL }}" method="POST" name="sentMessage" id="{{ $data.formName | default "contactForm" }}">
+          <form class="contact" action="{{ $contact.postURL }}" method="POST" name="sentMessage" id="{{ $contact.formName | default "contactForm" }}">
         {{ end }}
             <div class="row py-3">
               <div class="col-md-6">
-                {{ with $data.fields.name }}
+                {{ with $contact.fields.name }}
                   <div class="form-group">
                     <input
                       id="name"
                       name="name"
                       class="form-control"
                       type="text"
-                      data-validation-error-msg="{{ .error | default (i18n "contact.defaultNameError") }}"
-                      data-validation-error-msg-container="#name-error"
+                      contact-validation-error-msg="{{ .error | default (i18n "contact.defaultNameError") }}"
+                      contact-validation-error-msg-container="#name-error"
```

Comment:
This change breaks the contact form. Search and replace did replace a bit too much here ;)